src/video/uikit/SDL_uikitmetalview.m

```diff
@@ -79,16 +79,24 @@ SDL_MetalView UIKit_Metal_CreateView(SDL_VideoDevice *_this, SDL_Window *window)
         CGFloat scale = 1.0;
         SDL_uikitmetalview *metalview;
 
-#ifndef SDL_PLATFORM_VISIONOS
         if (window->flags & SDL_WINDOW_HIGH_PIXEL_DENSITY) {
             /* Set the scale to the natural scale factor of the screen - then
              * the backing dimensions of the Metal view will match the pixel
              * dimensions of the screen rather than the dimensions in points
              * yielding high resolution on retine displays.
              */
+#ifndef SDL_PLATFORM_VISIONOS
```

Comment:
Can you flip these so the `#ifdef SDL_PLATFORM_VISIONOS` is first?

Reply:
I could but I would probably keep it mainly because vision os is a niche platform piggy backing this code of all other more mature iOS ones which maybe should come first 😄 
But if you guys prefer the other way around sure!

Reply:
It looks like what you have is consistent with the other VISIONOS checks, so let's keep it. Thanks!